Makefile

```diff
@@ -102,35 +102,28 @@ fix-lint: ## fix lint issues using autoflake, autopep8, and isort
 
 # TEST TARGETS
 
-.PHONY: test-unit
-test-unit: ## run unit tests using pytest
-	python -m pytest --disable-warnings --cov=copulas tests/unit
+.PHONY: test-end-to-end
+test-end-to-end: ## run tests quickly with the default Python
```

Comment:
can we update the comment to say what it actually does? Is it running integration tests only?